docs/WalletGuide.md

```diff
@@ -0,0 +1,301 @@
+# Wallet SDK usage guide
+
+<!--- TOC -->
+
+* [Getting started](#getting-started)
+  * [Configuring client](#configuring-client)
+* [Anchor](#anchor)
+  * [Add client domain signing](#add-client-domain-signing)
+    * [Example](#example)
+* [Build on Stellar](#build-on-stellar)
+  * [Account service](#account-service)
+  * [Transaction builder](#transaction-builder)
+* [Recovery](#recovery)
+
+<!--- END -->
+
+## Getting Started
+
+First, a root Wallet object should be created. This is a core class, that provides all functionality available in the current SDK. Later, it will be shown, how to use the wallet object to access methods.
+
+It's advised to have a singleton wallet object shared across the app.  
+Creating a wallet with default configuration connected to testnet is simple:
+
+```typescript
+let wallet = Wallet.TestNet();
+```
+
+The wallet instance can be further configured. For example, to connect to the public main network:
+
+```typescript
+let wallet = new Wallet({
+  stellarConfiguration: StellarConfiguration.MainNet(),
+});
+```
+
+### Configuring client
+
+The Wallet SDK uses the [axios client](https://axios-http.com/docs/intro) for all network requests. A default client is used by default, or a custom client can given and configured like so:
+
+```typescript
+const customClient: AxiosInstance = axios.create({
+  baseURL: "https://some-url.com/api",
+  timeout: 1000,
+  headers: { "X-Custom-Header": "foobar" },
+});
+let appConfig = new ApplicationConfiguration(
+  DefaultSigner,
+  customClient
+);
+let wal = new Wallet({
+  stellarConfiguration: StellarConfiguration.TestNet(),
+  applicationConfiguration: appConfig,
+});
+  ```
+
+## Anchor
+
+Build on and off ramps with anchors for deposits and withdrawals.
+
+```typescript
+let anchor = wallet.anchor({ homeDomain: "testanchor.stellar.org" });
```

Comment:
Will url work or does it need to be strictly domain?

Reply:
url works too